en/docs/concepts/authentication/mfa-with-fido.md

```diff
@@ -0,0 +1,71 @@
+# Fast IDentity Online (FIDO)
+
+## What is FIDO?
+
+Fast IDentity Online (FIDO) is a set of protocols that supports authentication of any type including fingerprint, biometrics, security tokens etc. FIDO specifications support [multi-factor authentication](../multi-factor-authentication) and makes it simpler to authenticate users while also ensuring that security is enhanced. 
+
+FIDO protocols are based on public key cryptography and are strongly resistant to phishing. In order to address a wide range of use cases and deployment scenarios, FIDO provides two specifications that support two different user experience paths:
+
+- Universal Authentication Framework (UAF)
+- Universal Second Factor (U2F)
+
+![fido](../../assets/img/concepts/fido.png)
+
+---
+
+## UAF 
+
+Universal Authentication Framework (UAF) involves a passwordless experience with the following key processes:
+
+-   The user carries the client device with the UAF stack installed. 
+-   The user presents a local biometric or PIN.
+-   The website can choose whether to retain the password.
```

Comment:
Saying that UAF involves a passwordless experience in the first sentence and saying "The website can choose whether to retain the password" can be confusing to a new user. WDYT?